src/couch/src/couch_debug.erl

```diff
@@ -891,6 +904,23 @@ print_table(Rows, TableSpec) ->
     ),
     ok.
 
+print_report(Report) ->
+    TableSpec = [
+        {50, centre, info},
```

Comment:
For consistency with other functions in this module can we use `left` here?

- https://github.com/apache/couchdb/blob/main/src/couch/src/couch_debug.erl#L622
- https://github.com/apache/couchdb/blob/main/src/couch/src/couch_debug.erl#L803

Should we update mem_hoggers functions to use `left` as well (in a separate PR)?

I don't have strong opinion on whether we should fix it as well as any preference `left` vs `centre`. I just prefer it to be consistent.